src/TaskQueue.js

```diff
@@ -0,0 +1,43 @@
+const iterate = require('taskcluster-lib-iterate');
+const assert = require('assert');
+class TaskQueue {
+  constructor(queue, type) {
+    assert(queue, 'Instance of taskcluster queue is required');
+    this.queue = queue;
+    this.workerType = type;
+    this.provisionerId = 'garbage-hybrid1999';
+    this.workerGroup = 'garbage-hybrid1999';
```

Comment:
These two values should still come from `cfg` -- we don't want to use `garbage-hybrid1999` in production :)